### ISSUE TITLE
Manage global secrets in the UI

### ISSUE THREAD

**Issue by ptagr** [drone/drone#2106]
Drone version: `0.7.3`
This would allow us to add new global secrets to a running server

**Comment by donny-dont** [comment]
Global secrets are a enterprise feature. Additionally there is a cli specific repo as this has been split into its own project.

**Comment by tboerger** [comment]
And again, please follow the issue template as already mentioned on your other request.

**Comment by bradrydzewski** [comment]
@punitag global secrets are defined in a file and are therefore static (cannot be dynamically added or removed). We will consider changing this behavior in a future release and allowing sysadmins to manage secrets via the user interface.

In the mean time, if you want to dynatmically manage global secrets may consider the vault integration? http://docs.drone.io/vault-secrets/

**Comment by rochdev** [comment]
@bradrydzewski Do global secrets support Vault however?

**Comment by bradrydzewski** [comment]
Global secrets refers to secrets stored directly in drone. This issue requests they are managed int he user interface instead of a file.
http://docs.drone.io/global-secrets/

Vault secrets is a separate feature where drone can read the secrets from vault datastore. It is not related to global secrets.
http://docs.drone.io/vault-secrets/

**Comment by qrpike** [comment]
@bradrydzewski @donny-dont  When did global secrets become enterprise only? I'm using 0.5 with global secrets now. Upgrading will break that?

**Comment by bweston92** [comment]
@qrpike yes.

**Comment by delfer** [comment]
```sql
DELETE FROM secrets WHERE secret_repo_id!=120;
INSERT INTO secrets (secret_repo_id, secret_name, secret_value,secret_images,secret_events,secret_skip_verify,secret_conceal) 
  SELECT repo_id, secret_name, secret_value,secret_images,secret_events,secret_skip_verify,secret_conceal 
  FROM repos, secrets 
WHERE repo_active AND repo_id != 120 AND secret_repo_id=120;
```
global secrets 4 all ;-)

**Comment by qrpike** [comment]
@delfer much appreciated!

**Comment by bradrydzewski** [comment]
closed by 96132e3d0a625a63dfff3ac77ecd21c0245f1775
